WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -38,6 +38,12 @@ function sortedDelegates(delegates) {
 }
 
 function DelegatesOfRegion({ activeSeniorDelegate, delegates, isAdminMode }) {
+  const delegatesOfRegion = React.useMemo(() => sortedDelegates([
+    ...delegates.filter(
+      (user) => user.region_id === activeSeniorDelegate.region_id
+        && (user.delegate_status !== 'trainee_delegate' || isAdminMode),
+    ),
+  ]), [activeSeniorDelegate, delegates, isAdminMode]);
```

Comment:
```suggestion
  const delegatesOfRegion = React.useMemo(() => sortedDelegates(
    delegates.filter(
      (user) => user.region_id === activeSeniorDelegate.region_id
        && (user.delegate_status !== 'trainee_delegate' || isAdminMode),
    ),
  ), [activeSeniorDelegate, delegates, isAdminMode]);
```
and renaming the helper to `sortDelegates` would be nice.

Reply:
Done, thank you